cognite/client/_api/three_d.py

```diff
@@ -378,7 +378,14 @@ def update_thumbnail(self, model_id: int, revision_id: int, file_id: int) -> Non
         self._post(resource_path, json=body)
 
     def list_nodes(
-        self, model_id: int, revision_id: int, node_id: int = None, depth: int = None, limit: int = 25
+        self,
+        model_id: int,
+        revision_id: int,
+        node_id: int = None,
+        depth: int = None,
+        limit: int = 25,
```

Comment:
I think we tend to keep limit as the last parameter, as it makes the docs a little more readable